CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (gov) [#16098](https://github.com/cosmos/cosmos-sdk/pull/16098) Improve gov's proposal message processing by remoing duplicate message check
```

Comment:
We do not need a changelog, this extra check has never been released (happened during the removal of validate basic checks).

Reply:
i see... 😆